maskrcnn_benchmark/config/defaults.py

```diff
@@ -272,3 +272,5 @@
 _C.OUTPUT_DIR = "."
 
 _C.PATHS_CATALOG = os.path.join(os.path.dirname(__file__), "paths_catalog.py")
+
+_C.TENSORBOARD_EXP_NAME = "maskrcnn-benchmark"  # tensorboard experiment name
```

Comment:
I still think that there is no need for two arguments: if you run two trainings on the same directory, it will have conflicts with the checkpoints and it will not work as expected.

In this case, I would just pass the same `OUTPUT_DIR` here.

Reply:
It will not conflict because I add a time stamp. 

What is your idea? To have each tensorboard log in the model directory instead of a dedicated logs folder?

Reply:
What I meant is that if you try to run multiple training experiments in the same `OUTPUT_DIR` folder, then you'll have problems during training, as the `Checkpointer` might pick the checkpoint from a different experiment.

If you would want to have all the TensorBoard logs in a single place (because it allows to compare different experiments), then I'd keep an extra argument in the config (or the training script) with the full path, and you can add the timestamp in your logic.

Reply:
that sounds better!

Reply:
Can't the tensorboard logs all be in the OUTPUT_DIR instead of a separate logs directory, but still be compared? This is how the unofficial pytorch detectron [handles it](https://github.com/roytseng-tw/Detectron.pytorch/blob/9defe952cb8b1a4b90b1c1ad19eb5b3614b01418/tools/train_net.py#L301). As long as all the model directories are in the same root directory, e.g.
`experiments/model1/`, `experiments/model2/`, `experiments/model3/`, each containing tensorboard summaries, pointing tensorboard to the `experiments` directory will show all the logs.

Reply:
admitting that every model has the same root output dir, yes.